microsoft-edge/dualengine/reference/accelerator.md

```diff
@@ -25,7 +25,7 @@ api_location:
 
 > [!IMPORTANT]
 > The DualEngine API is part of a Limited Access Feature (see [LimitedAccessFeatures class](/uwp/api/windows.applicationmodel.limitedaccessfeatures)). For more information or 
-> to request an unlock token, contact [Microsoft Support](https://support.serviceshub.microsoft.com/supportforbusiness/create?sapId=d15d3aa2-0512-7cb8-1df9-86221f5cbfde).
+> to request an unlock token, contact [Microsoft Support](https://go.microsoft.com/fwlink/?linkid=2271232&clcid=0x409).

```

Comment:
```suggestion
> to request an unlock token, contact [Microsoft Support](https://go.microsoft.com/fwlink/?linkid=2271232).
```
Global in PR: remove &clcid=0x409 from this link
@captainbrosset 

Test:
https://go.microsoft.com/fwlink/?linkid=2271232&clcid=0x409
https://go.microsoft.com/fwlink/?linkid=2271232

Reply:
This is a timely request so I'll merge as is, and open a new issue tro track removing that part of the URL.